auditlog/mixins.py

```diff
@@ -99,7 +101,7 @@ def msg(self, obj):
                     format_html(
                         "<tr><td>{}</td><td>{}</td><td>{}</td><td>{}</td></tr>",
                         i,
-                        field,
+                        obj.verbose_name(field),
```

Comment:
Do we have any tests for that?

Reply:
yes there is already a test for that ("test_changes_msg_m2m")